test/PerfTest/WebApiPerformance.Service/WebApiPerformance.Service.csproj

```diff
@@ -26,16 +26,16 @@
   <Choose>
     <When Condition=" '$(TestType)'=='Private' ">
       <ItemGroup>
-        <Reference Include="Microsoft.OData.Core, Version=7.3.1.10814, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-          <HintPath>..\..\sln\packages\Microsoft.OData.Core.7.3.1\lib\portable-net45+win8+wpa81\Microsoft.OData.Core.dll</HintPath>
+        <Reference Include="Microsoft.OData.Core, Version=7.5.0.20201, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+          <HintPath>..\..\..\sln\packages\Microsoft.OData.Core.7.5.0-Nightly201802011100\lib\portable-net45+win8+wpa81\Microsoft.OData.Core.dll</HintPath>
           <Private>True</Private>
         </Reference>
-        <Reference Include="Microsoft.OData.Edm, Version=7.3.1.10814, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-          <HintPath>..\..\sln\packages\Microsoft.OData.Edm.7.3.1\lib\portable-net45+win8+wpa81\Microsoft.OData.Edm.dll</HintPath>
+        <Reference Include="Microsoft.OData.Edm, Version=7.5.0.20201, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+          <HintPath>..\..\..\sln\packages\Microsoft.OData.Edm.7.5.0-Nightly201802011100\lib\portable-net45+win8+wpa81\Microsoft.OData.Edm.dll</HintPath>
           <Private>True</Private>
         </Reference>
-        <Reference Include="Microsoft.Spatial, Version=7.3.1.10814, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-          <HintPath>..\..\sln\packages\Microsoft.Spatial.7.3.1\lib\portable-net45+win8+wpa81\Microsoft.Spatial.dll</HintPath>
+        <Reference Include="Microsoft.Spatial, Version=7.5.0.20201, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+          <HintPath>..\..\..\sln\packages\Microsoft.Spatial.7.5.0-Nightly201802011100\lib\portable-net45+win8+wpa81\Microsoft.Spatial.dll</HintPath>
```

Comment:
use the nightly build still?

Reply:
good catch! forgot to update the WebApiPerf solution.